java-symbol-solver-core/src/main/java/me/tomassetti/symbolsolver/javaparser/Navigator.java

```diff
@@ -23,8 +23,34 @@ private Navigator() {
         if (cu.getTypes() == null) {
             return Optional.empty();
         }
-        return cu.getTypes().stream().filter((t) -> t.getName().equals(name)).findFirst();
+        
+        String[] nameParts = name.split("\\.", 2);
```

Comment:
This name splitting seems to be repeated in several points, maybe we want to create an helper method for that.